### DISCUSSION TITLE
hosts

### DISCUSSION THREAD

**Discussion by Reycans** [ltsp/ltsp#441] [Q&A] [answered]
Hello. I need to add an entry to the file / etc / hosts and transfer it to clients, how can this be done?

**Comment by enaut** [comment, marked as answer]
See `HOSTS_x` in ltsp.conf: https://ltsp.org/man/ltsp.conf/#parameters

basically add lines to the ltsp.conf file like this (in the clients section):
```bash
[clients]
HOSTS_x="192.168.67.10 nfs-server"
HOSTS_y="192.168.67.12 git.server"
```
